### ISSUE TITLE
Create Quest 3 LookSee version

### ISSUE THREAD

**Issue by tomgoddard** [RBVI/LookSee#7]
Want to release a LookSee version for Quest 3 which came out this week.  Current LookSee v9 targets Quest 2 and Quest Pro.

**Comment by tomgoddard** [comment]
Updated Unity Oculus Integration package from version 53.1 to 57.0.
Need Oculus XR 4.1 package for Quest 3 support.
That required updating Unity from version 2022.2 that LookSee has been using to 2023.1.16.
With new Unity the TextMeshPro v3.0.6 package used for buttons and labels no longer worked due to an error with 0 size textures.  That required and update to 3.2.0 from 2021 and 2022 which is only available as prereleases.  It never became an official release and the next Unity 2023.2 is eliminating it and folding it into a new uGUI package based on 3.2.0-pre4.  I tried 3.2.0-pre4 and worked to remedy severe aliasing of text for a few hours which was fixed by updating to 3.2.0-pre6.

With these updates I was able to build a Quest 3 targeted LookSee.  Don't have Quest 3 yet to test on although we ordered one yesterday.  Tested on Quest 2 and worked without any new issues.
